meson.build

```diff
@@ -1811,7 +1811,9 @@ if have and efi_arch == 'x64' and cc.links('''
         efi_cpu_family_alt = 'x86'
 endif
 
-want_ukify = get_option('ukify').require(python_39, error_message : 'Python >= 3.9 required').allowed()
+pefile = pymod.find_installation('python3', required: false, modules : ['pefile'])
+
+want_ukify = get_option('ukify').require(python_39 and (want_tests != 'true' or pefile.found()), error_message : 'Python >= 3.9 and pefile required').allowed()
```

Comment:
Why missing pefile module is allowed when test is disabled?
Not familiar with python code, but ukify.py imports pefile.

If the condition about testing can be dropped, maybe,
```
pefile = pymod.find_installation(..., required: get_option('ukify'), ...)
```
is better?

Reply:
That would make it required even when tests are disabled, which I explicitly wanted to avoid, as the build dependency is only needed in that case. Unfortunately find_installation's 'required' does not support logical expressions, only a get_option can be passed, otherwise it errors out.

Reply:
The dependency check is only useful for things needed at build time